src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -0,0 +1,137 @@
+    [rn/view {:style style/container}
+     [account-switcher/view
+      {:on-press      navigation/navigate-back
+       :icon-name     :i/arrow-left
+       :margin-top    (safe-area/get-top)
+       :switcher-type :select-account}]
+     [rn/view {:style style/inputs-container}
+      [quo/swap-input
+       {:type                 :pay
+        :error?               false
+        :token                pay-token-symbol
+        :customization-color  :blue
+        :show-approval-label? false
+        :status               :default
+        :currency-symbol      currency-symbol
+        :on-swap-press        #(js/alert "Swap Pressed")
+        :on-token-press       #(js/alert "Token Pressed")
+        :on-max-press         #(js/alert "Max Pressed")
+        :value                pay-value
+        :fiat-value           receive-token-fiat-value
+        :network-tag-props    {:title    (str (i18n/label :t/max {:number 200}) " " pay-token-symbol)
+                               :networks [{:source (:source network)}]}
+        :approval-label-props {:status              :approve
+                               :token-value         pay-value
+                               :button-props        {:on-press
+                                                     #(js/alert "Approve Pressed")}
+                               :customization-color color
+                               :token-symbol        pay-token-symbol}}]
+      [quo/swap-order-button
+       {:container-style style/swap-order-button
+        :on-press        #(js/alert "Pressed")}]
+      [quo/swap-input
+       {:type                 :receive
+        :error?               false
+        :token                receive-token-symbol
+        :customization-color  color
+        :show-approval-label? false
+        :enable-swap?         true
+        :status               :default
+        :currency-symbol      currency-symbol
+        :on-swap-press        #(js/alert "Swap Pressed")
+        :on-token-press       #(js/alert "Token Pressed")
+        :on-max-press         #(js/alert "Max Pressed")
+        :value                receive-amount
+        :fiat-value           pay-token-fiat-value
+        :container-style      style/receive-token-swap-input-container}]]
+     [rn/view {:style style/footer-container}
+      (when swap-proposal
+        [transaction-details
+         {:native-currency-symbol native-currency-symbol
+          :max-slippage           max-slippage
+          :loading-fees?          loading-fees?}])
+      [quo/bottom-actions
+       {:actions          :one-action
+        :button-one-label (i18n/label :t/review-swap)
+        :button-one-props {:disabled?           (or (not swap-proposal)
+                                                    loading-fees?)
+                           :customization-color color
+                           :on-press            #(js/alert "Review swap pressed")}}]]
+     [:<>
```

Comment:
is this needed?

Reply:
Good catch, it is not needed actually, will remove